pkg/quota/evaluator/core/persistent_volume_claims.go

```diff
@@ -119,8 +124,8 @@ func (p *pvcEvaluator) Handles(a admission.Attributes) bool {
 }
 
 // Matches returns true if the evaluator matches the specified quota with the provided input item
-func (p *pvcEvaluator) Matches(resourceQuota *api.ResourceQuota, item runtime.Object) (bool, error) {
-	return generic.Matches(resourceQuota, item, p.MatchingResources, generic.MatchesNoScopeFunc)
+func (p *pvcEvaluator) Matches(resourceQuota *api.ResourceQuota, item runtime.Object, matchPriorityScope bool) (bool, error) {
```

Comment:
also not obvious to me yet why the additional bool is needed on Matches func calls.